### DISCUSSION TITLE
How to increase the sidebar width?

### DISCUSSION THREAD

**Discussion by dfalbel** [quarto-dev/quarto-cli#1198] [Q&A] [answered]
We have a Quarto website with a `style: "docked"` sidebar which we find too narrow for the items titles. 
I have found that the following does what we want but adds extra margin to the mobile view which we don't really want.

```
/*-- scss:mixins --*/
$grid-screen-inset-padding: 1.5em;
```

I wonder what's would be the best way to modify the sidebar width.

Here's the current behavior:
![Current behavior](https://user-images.githubusercontent.com/4706822/174874932-bc283107-72c8-488d-92f4-fcb91ac54980.png)

And the desired:

![image](https://user-images.githubusercontent.com/4706822/174875135-18ee1222-2673-4918-bcf6-bb72e2679708.png)

**Comment by sumnerg** [comment]
Would like to see an answer to this too.

**Reply by pat-s** [reply]
@sumnerg

Please upvote the initial Q instead of posting an answer that just repeats the question :)

**Comment by paniterka** [comment, marked as answer]
Hi all, it looks like this useful functionality [has been implemented in Quarto 1.3+](https://quarto.org/docs/prerelease/1.3.html). I have a pre-release 1.3.75 running and have been able to change the sidebar width in `_quarto.yml` using 
```
format:
  html:
    grid:
      sidebar-width: 500px
```

**Reply by mcanouil** [reply]
Indeed, see #3664, #2509 and #3198 amongst other discussion.

**Reply by cavo789** [reply]
Great! I was searching for this option since one hour...

**Reply by mcanouil** [reply]
> Great! I was searching for this option since one hour...

Using quarto.org search? It's the second hit with "sidebar width"

**Reply by cavo789** [reply]
On Google ;-{  
Then I've tried IA (Gemini) which told me I should create a custom CSS (ok, let's try) but didn't work.  I was able to change the width of the sidebar by using a rule for `#quarto-sidebar` but the content was then put "below" the increased sidebar and I was then trying to search for the other classes to change so really wrong direction.

**Reply by mcanouil** [reply]
Before web search or way before AI and before posting, searching in software's documentation is the most efficient way to get answers.
